ideascube/serveradmin/catalog.py

```diff
@@ -154,8 +154,8 @@ def commit(self):
         for libpath in glob(os.path.join(libdir, '*.xml')):
             zimname = os.path.basename(libpath)[:-4]
 
-            with open(libpath, 'r') as f:
-                et = etree.parse(f)
+            with open(libpath, mode='r', encoding='utf-8') as f:
+                et = etree.fromstring(f.read())
```

Comment:
Is it necessary to do the read ourselves ?
etree.parse is supposed to use the read method anyway and maybe in a more efficient way (do not read all in once)